doc/users/next_whats_new/new_color_cycle.rst

```diff
@@ -3,3 +3,9 @@ New style colorblind-friendly color cycle
 
 A new style defining a color cycle has been added, tableau-colorblind10, to provide another option for 
 colorblind-friendly plots.
+
+Cividis colormap
```

Comment:
I think this change should get its own file, not be put in its own.

it'd be cool if you included a plot in here so people could see what it looks like...

```rst
.. plot::

    import matplotlib.pyplot
    import numpy as np

    fig, ax = plt.subplots()
    pcm = ax.pcolormesh(np.rand.random(32,32), cmap='cividis')
    fig.colorbar(pcm)
```